System.IO.Abstractions/IFileSystem.cs

```diff
@@ -5,6 +5,7 @@ public interface IFileSystem
         FileBase File { get; }
         DirectoryBase Directory { get; }
         IFileInfoFactory FileInfo { get; }
+        IFileStreamFactory FileStream { get; }
```

Comment:
This is my only concern. Since we're adding onto IFileSystem, it's a breaking change.. but is anyone really writing their own classes using IFileSystem?

Reply:
Yeah, same goes for 07227bca6b82839b33cb863e971c116b61c8a3d6... I also don't think this will be a big problem for people. Whenever we are done with the old issues/PRs we should again think about release strategy, versioning, issue tagging, etc...

Reply:
To be clear. I would merge this one and as a follow-up take care about our future strategy.

Reply:
Yeah, I would agree. It's probably an incredibly small subset of users it would impact.